src/Middleware/Reauthenticate.php

```diff
@@ -2,34 +2,12 @@
 
 namespace Mpociot\Reauthenticate\Middleware;
 
-use Carbon\Carbon;
 use Closure;
+use Mpociot\Reauthenticate\ReauthLimiter;
 
 class Reauthenticate
 {
     /**
-     * Number of minutes a successful Reauthentication is valid.
-     *
-     * @var int
-     */
-    protected $reauthTime = 30;
-
-    /**
-     * Validate a reauthenticated Session data.
-     *
-     * @param \Illuminate\Session\Store $session
-     *
-     * @return bool
-     */
-    private function validAuth($session)
-    {
-        $validationTime = Carbon::createFromTimestamp($session->get('reauthenticate.life', 0));
-
-        return $session->get('reauthenticate.authenticated', false) &&
-            ($validationTime->diffInMinutes() <= $this->reauthTime);
-    }
```

Comment:
Main issue with extending this package on other package is that this method is marked as `private`, therefore if I need to use different routes to handle reauthentication I would need to override the whole class (instead of just the redirection). Of course this also solved via #15